test/README.md

```diff
@@ -8,29 +8,15 @@ arguments.
 
 ## Running tests
 
-To run the tests, execute the `run` script with root priviledges:
+To run the tests, execute the `run` script with root privileges:
 
-`sudo ./run`
+```bash
+sudo ./run
+```
 
-Specifically for the snap installation, it’s necessary to create some
-manual aliases:
+For testing the snap, it’s necessary to remove the Debian package (if installed) and alias the snapped applications beforehand:
```

Comment:
It's a good practice to keep each line under 80 characters:

```suggestion
For testing the snap, it’s necessary to remove the Debian package (if installed)
and alias the snapped applications beforehand:
```

Reply:
Why is that a good practice? This dates back to 1960 and from hardware limitations in punch cards. Nowadays the monitors are wide and there is word wrapping in most editors for extra long lines. Why should it still be 80 and not make full use of the display width? 

Moreover, the so-called rule is for code, not for text in readme. Making such a limit makes it ultra tedious to make changes to text.

Reply:
It makes it difficult to analyze code/text in some scenarios, such as in code reviews or when viewing Markdown text in its raw form. Usually, GitHub doesn't perform word wrapping, as seen in [this case](https://github.com/canonical/chip-tool-snap/blob/49ed797d93b6ba556a47c74c397ca4a8f5dfb5f3/README.md?plain=1#L30), for example.

I agree that it's an old convention, but in some scenarios, it's still useful to avoid having to pay attention to dragging the bar with the mouse to continue reading.